app/src/common/utils/timeDateUtils.test.js

```diff
@@ -0,0 +1,91 @@
+import { getDuration, approximateTimeFormat, dateFormat, daysBetween } from './timeDateUtils';
+
+const NOW = Date.now();
+const ONE_SECOND = 1000;
+const ONE_MINUTE = 60 * 1000;
+const ONE_HOUR = 3600 * 1000;
+const ONE_DAY = 24 * 3600 * 1000;
+
+describe('getDuration', () => {
+  test('should return sting', () => {
+    expect(typeof getDuration(NOW, NOW + ONE_HOUR)).toBe('string');
+  });
+
+  test('should return zero seconds', () => {
+    expect(getDuration(NOW, NOW)).toBe('0s');
+  });
+
+  test('should contain seconds', () => {
```

Comment:
Should contain seconds in which case?
The same question also applies to next tests